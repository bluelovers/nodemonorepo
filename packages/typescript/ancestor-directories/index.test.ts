import subject from './index'

it('matches snapshot', () => {
  expect(
    [
      'abc/def/ghi',
      './abc/def/ghi',
      '/abc/def/ghi',
      'abc',
      '.',
      ''
    ].map(
      x => ({
        input: x,
        output: Array.from(subject(x))
      })
    )
  ).toMatchSnapshot()
})
