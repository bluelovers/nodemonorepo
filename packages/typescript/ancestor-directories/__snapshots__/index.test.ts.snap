// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
Array [
  Object {
    "input": "abc/def/ghi",
    "output": Array [
      "abc/def/ghi",
      "abc/def",
      "abc",
      ".",
    ],
  },
  Object {
    "input": "./abc/def/ghi",
    "output": Array [
      "./abc/def/ghi",
      "./abc/def",
      "./abc",
      ".",
    ],
  },
  Object {
    "input": "/abc/def/ghi",
    "output": Array [
      "/abc/def/ghi",
      "/abc/def",
      "/abc",
      "/",
    ],
  },
  Object {
    "input": "abc",
    "output": Array [
      "abc",
      ".",
    ],
  },
  Object {
    "input": ".",
    "output": Array [
      ".",
    ],
  },
  Object {
    "input": "",
    "output": Array [
      "",
      ".",
    ],
  },
]
`;
